src/app/Library/CrudPanel/Traits/Input.php

```diff
@@ -49,6 +49,7 @@ private function getDirectInputsFromInput($input, $model = false, $fields = [],
 
         $input = $this->decodeJsonCastedAttributes($input, $model);
         $input = $this->compactFakeFields($input, $model, $fields);
+        $input = $this->addMorphToInputsFromRelationship($input);
```

Comment:
Perhaps a better name for this would be "include" if the next method is "exclude"?

```suggestion
        $input = $this->includeMorphToInputsFromRelationship($input);
```